src/libs/db2/migrations/20180523131528_create-user_account.js

```diff
@@ -32,7 +32,7 @@ exports.up = async knex =>
     t.text('email').notNull();
     t.text('phone_number');
     t.boolean('pia_seen').notNull().defaultTo(false);
-    t.boolean('active').notNull().defaultTo(false);
+    t.boolean('active').notNull().defaultTo(true);
```

Comment:
This won't take effect unless you drop and re-create the database. You can create a new migration that just changes the default: `201903042343_change_active_default.js`